.status

```diff
@@ -6,10 +6,10 @@ test/*: SkipByDesign # Testing in build to get a package root.
 
 # Prevent execution of non-test files with name *_test.dart in packages.
 build/test/packages: SkipByDesign
-build/test/mock_tests/packages: SkipByDesign
+build/test/packages: SkipByDesign
```

Comment:
Is this not the same line twice?

Reply:
Indeed! They used to be different (and the `build/test/packages` line used to have no effect because there were no tests in `test`, only in `test/mock_tests`); I'm removing the redundant line in the next CL.